src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/help-disableNotifications.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Whether to notify GitHub with the build status when the build starts and the build finishes.
```

Comment:
There is also a notification when the build is _scheduled_, but that is probably too much detail for this help anyway.